app/test/process_proposal_test.go

```diff
@@ -203,18 +206,42 @@ func TestProcessProposal(t *testing.T) {
 			expectedResult: abci.ResponseProcessProposal_REJECT,
 		},
 		{
-			name:  "invalid namespace",
+			name:  "invalid blob namespace",
 			input: validData(),
 			mutator: func(d *core.Data) {
 				d.Blobs[0] = core.Blob{
 					NamespaceId:      invalidNamespace.ID,
 					Data:             data,
-					NamespaceVersion: uint32(invalidNamespace.Version),
 					ShareVersion:     uint32(appconsts.ShareVersionZero),
+					NamespaceVersion: uint32(invalidNamespace.Version),
 				}
 			},
 			expectedResult: abci.ResponseProcessProposal_REJECT,
 		},
+		{
+			name:  "invalid namespace in index wrapper tx",
```

Comment:
[Question]
Is the invalid namespace == namespace version 1?

Reply:
yep! Specifically https://github.com/rootulp/celestia-app/blob/ceeb7f45bef4d30ffc94760a48c78452505b4d08/testutil/blobfactory/payforblob_factory.go#L408